### PR TITLE
Standard drop down friendly name fixed

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [x] Sanity Tests are successfully executed for New and Existing Functionality
- [x] Verify that changes are compatible with all relevant browsers and platforms.
- [x] After creating pull request there should not be any conflicts
- [x] Resolve all Codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Update the Help Library document to match any feature changes

## Summary by CodeRabbit

- **New Features**
	- Enhanced accessibility testing capabilities with the addition of a comprehensive "All" option.
- **Refactor**
	- Improved handling of enumeration types in the user interface components, ensuring more flexible configuration options.

### CODE REVIEW

File: Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs [hunk 1]
```diff
@@ -91,7 +91,7 @@ public ActAccessibilityTestingEditPage(ActAccessibilityTesting act)
                 }
             }
             xStdCB.ItemsSource = mAct.Items;
-            xStdCB.Init(mAct, nameof(mAct.StandardList));
+            xStdCB.Init(mAct, nameof(mAct.StandardList),true);
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs:95]: Added

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 2]
```diff
@@ -44,6 +45,7 @@ public partial class MultiSelectComboBox : UserControl
         private ObservableCollection<Node> _nodeList;
         private object obj;
         private string AttrName;
+        private bool isenum = true;
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 3]
```diff
@@ -170,15 +172,32 @@
...
+                    foreach (Node s in _nodeList)
+                    {
+                        if (s.IsSelected && s.Title != "All")
+                            _selectedCount++;
+                    }
+                    if (_selectedCount == _nodeList.Count - 1)
+                        _nodeList.FirstOrDefault(i => i.Title == "All").IsSelected = true;
+                    else
+                        _nodeList.FirstOrDefault(i => i.Title == "All").IsSelected = false;
+                }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:198]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 4]
```diff
@@ -206,34 +235,61 @@
...
+                                SelectedItems.Add(new OperationValues() { Value = node.Title, Guid = guidOutput });
+                                temp.Add(new OperationValues() { Value = node.Title, Guid = guidOutput });
+                            }
+                            else
+                            {
+                                SelectedItems.Add(new OperationValues() { Value = node.Title });
+                                temp.Add(new OperationValues() { Value = node.Title });
+                            }
                         }
+
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:279]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 5]
```diff
@@ -192,9 +211,19 @@
...
+                    Node node = _nodeList.FirstOrDefault(i => i._title == keyValue.Value);
+                    if (node != null)
+                        node.IsSelected = true;
+                }
+                else
+                {
+                    Node node = _nodeList.FirstOrDefault(i => i.Title == keyValue.Value);
+                    if (node != null)
+                        node.IsSelected = true;
+                }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:225]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 6]
```diff
@@ -170,15 +172,32 @@ private void CheckBox_Click(object sender, RoutedEventArgs e)
...
+                    foreach (Node s in _nodeList)
+                    {
+                        if (s.IsSelected && s.Title != "All")
+                            _selectedCount++;
+                    }
+                    if (_selectedCount == _nodeList.Count - 1)
+                        _nodeList.FirstOrDefault(i => i.Title == "All").IsSelected = true;
+                    else
+                        _nodeList.FirstOrDefault(i => i.Title == "All").IsSelected = false;
+                }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:198]: Handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 7]
```diff
@@ -206,34 +235,61 @@ private void SetSelectedItems()
...
+                                SelectedItems.Add(new OperationValues() { Value = node.Title, Guid = guidOutput });
+                                temp.Add(new OperationValues() { Value = node.Title, Guid = guidOutput });
+                            }
+                            else
+                            {
+                                SelectedItems.Add(new OperationValues() { Value = node.Title });
+                                temp.Add(new OperationValues() { Value = node.Title });
+                            }
                         }
+
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:279]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 8]
```diff
@@ -259,16 +315,33 @@ private void SetText()
...
+                        {
+                            displayText = new StringBuilder();
+                            displayText.Append("All");
+                            break;
+                        }
+                        else if (s.IsSelected == true && s.Title != "All")
+                        {
+                            displayText.Append(s.Title);
+                            displayText.Append(',');
+                        }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:344]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 9]
```diff
@@ -192,9 +211,19 @@ private void SelectNodes()
...
+                    Node node = _nodeList.FirstOrDefault(i => i._title == keyValue.Value);
+                    if (node != null)
+                        node.IsSelected = true;
+                }
+                else
+                {
+                    Node node = _nodeList.FirstOrDefault(i => i.Title == keyValue.Value);
+                    if (node != null)
+                        node.IsSelected = true;
+                }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:225]: handled

File: Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs [hunk 10]
```diff
@@ -44,6 +45,7 @@ public partial class MultiSelectComboBox : UserControl
         private ObservableCollection<Node> _nodeList;
         private object obj;
         private string AttrName;
+        private bool isenum = false;
```
Comment (Maheshkale447) [inline Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs:48]: Change name to ShowEnumDesc

File: Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs [hunk 11]
```diff
@@ -87,19 +87,20 @@ public ActAccessibilityTestingEditPage(ActAccessibilityTesting act)
...
                 if (!string.IsNullOrEmpty(StandaredTag.Value.ToString()))
                 {
-                    mAct.Items.Add(StandaredTag.Value.ToString(), StandaredTag.Value.ToString());
+                    mAct.Items.Add(StandaredTag.Value.ToString(), StandaredTag);
                 }
             }
             xStdCB.ItemsSource = mAct.Items;
-            xStdCB.Init(mAct, nameof(mAct.StandardList));
+            //Boolean value is to show description of enum value  
+            xStdCB.Init(mAct, nameof(mAct.StandardList),true);
```
Comment (Maheshkale447) [inline Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs:95]: xStdCB.Init(mAct, nameof(mAct.StandardList),ShowEnumDesc: true);

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs (hunk 11 above)
Comment (prashelke) [reply Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs]: Updated

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**